Arena/ArenaRPCs.cs

```diff
@@ -7,6 +7,57 @@ public static class ArenaRPCs
     {
 
         [RPCMethod]
+        public static void Arena_RemovePlayerWhoQuit(OnlinePlayer earlyQuitterOrLatecomer)
+        {
+            if (RainMeadow.isArenaMode(out var arena))
+            {
+                if (arena.arenaSittingOnlineOrder.Contains(earlyQuitterOrLatecomer.inLobbyId))
+                {
+                    arena.arenaSittingOnlineOrder.Remove(earlyQuitterOrLatecomer.inLobbyId); // you'll add them in NextLevel
+                }
+            }
+        }
+
+        [RPCMethod]
+        public static void Arena_AddPlayerWaiting(OnlinePlayer earlyQuitterOrLatecomer)
+        {
+            if (RainMeadow.isArenaMode(out var arena))
+            {
+                if (!arena.playersLateWaitingInLobbyForNextRound.Contains(earlyQuitterOrLatecomer.inLobbyId))
+                {
+                    arena.playersLateWaitingInLobbyForNextRound.Add(earlyQuitterOrLatecomer.inLobbyId); // you'll add them in NextLevel
+                }
+
+            }
+        }
+
+        [RPCMethod]
+        public static void Arena_NotifyRejoinAllowed(bool hasPermission)
+        {
+            var lobby = RWCustom.Custom.rainWorld.processManager.currentMainLoop as ArenaLobbyMenu;
+            if (RainMeadow.isArenaMode(out var arena))
+            {
+                if (lobby == null)
+                {
+                    RainMeadow.Debug("Could not start player");
+                    return;
+                }
+                RainMeadow.Debug("Starting game for player");
+                arena.isInGame = true; // state might be too late
+                arena.hasPermissionToRejoin = hasPermission;
+                RainMeadow.Debug("Start game immediately");
+                lobby.StartGame();
+            }
+        }
+
+
+        [RPCMethod]
+        public static void Arena_AddPlayerMidGame(OnlinePlayer newPlayer)
+        {
+            if (RainMeadow.isArenaMode(out var arena))
+            {
+                arena.arenaSittingOnlineOrder.Add(newPlayer.inLobbyId);
+

```

Comment:
```suggestion
            }
        }
```
these two braces are missing. Not sure if there's any other logic but might want to double check that everything in the PR is capable of being compiled with no errors and isn't missing something you intended to include.